plugins/org.jkiss.dbeaver.ext.db2/src/org/jkiss/dbeaver/ext/db2/DB2Utils.java

```diff
@@ -607,6 +607,24 @@ public static String formatSQLProcedureDDL(DB2DataSource db2DataSource, String r
 
         return result;
     }
+    
+    /**
+     * Retrieves the server variant information from the DB2 SQLCA.
+     */
+    public static char getServerVariant(DBRProgressMonitor monitor, JDBCSession session) throws SQLException {

```

Comment:
Please specify nullability annotations for the method's parameters.

Reply:
Are you saying that you'd like these annotated with `@NotNull`?